### ISSUE TITLE
registry() failed if there are values with REG_EXPAND_SZ type exist in Windows registry path

### ISSUE THREAD

**Issue by sergru** [PhilipSkinner/windows#5]
I need to find value with REG_EXPAND_SZ in key in Windows registry. Package function crashed with exception:

```
Exception: expandPath is not defined
stack: ReferenceError: expandPath is not defined
    at RegType.parse (PROJECT_PATH\node_modules\windows\lib\registry.js:114:25)
    at new Entry (PROJECT_PATH\node_modules\windows\lib\registry.js:259:72)
    at PROJECT_PATH\node_modules\windows\lib\registry.js:55:19
    at Array.reduce (<anonymous>)
    at registry (PROJECT_PATH\node_modules\windows\lib\registry.js:35:21)
    at getV (PROJECT_PATH\app.js:30:17)
```
I've check the 'windows' package, function expandPath() does not exist.

I think there is a typo in registry.js:114:25:
```
113:  REG_EXPAND_SZ: new RegType('REG_EXPAND_SZ', {
114:   parse: function(x){ return expandPath(x) }
115:  }),
```
 It should be just 'expand' based on line registry.js:7
```
7: var expand = require('./utility').expand;
```

Any help with it?

**Comment by PhilipSkinner** [comment]
Thanks for the bug report!

I've taken a look and you are correct, this should have been using the expand function to expand any environmental variables referenced within the value.

I've fixed this issue, and corrected a bug which happens if you attempted to read an empty REG_EXPAND_SZ  key.

I've released version 0.1.1, can you confirm this has fixed your issue?

**Comment by sergru** [comment]
Verified windows@0.1.1
Issue fixed.

Thanks for rapid response!
